tls/s2n_security_policies.c

```diff
@@ -20,6 +20,45 @@
 #include "tls/s2n_connection.h"
 #include "utils/s2n_safety.h"
 
+/* TLS1.2 default as of 05/24 */
+const struct s2n_security_policy security_policy_20240501 = {
+    .minimum_protocol_version = S2N_TLS12,
+    .cipher_preferences = &cipher_preferences_20240331,
```

Comment:
Not sure it really matters, but if we eventually intend to remove CBC, should we leave out `TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384`? If clients have been fine so far, it might be worth keeping it disabled to allow removing CBC to be slightly less breaking in the future.

Reply:
I'm not a fan of the inconsistency. I'd have to specifically create new cipher preferences with that same inconsistency / bug. And I don't think missing one CBC option makes it any less breaking.